stan/math/rev/core/vari.hpp

```diff
@@ -94,11 +94,11 @@ class vari_value<T, require_floating_point_t<T>> : public vari_base {
   /**
    * Rows at compile time
    */
-  static constexpr int RowsAtCompileTime{1};
+  static constexpr int RowsAtCompileTime{0};
```

Comment:
I changed this from 1 to 0 because if it's `1` then `is_row_vector` / `is_column_vector` would be `true` for `var` types which we want to be false.

Reply:
Do we even need these on scalars?

Reply:
Let me check. I think there was some code somewhere else that was failing because vari_value<double> didn't have these

Reply:
Yes we do since in `var_value<>` we grab the rows/cols at compile time from the `vari_type`. We can remove them in #2064 tho' because I break `var_value<double>` into it's own specialization